app-ios/Modules/Sources/Timetable/TimetableDayHeader.swift

```diff
@@ -8,40 +8,88 @@ struct TimetableDayHeader: View {
     let selectedDay: DroidKaigi2023Day
     let onSelect: (DroidKaigi2023Day) -> Void
 
+    // Define margin values to calculate holizontal position for capsule rectangle
+    private let buttonAreaLeadingMargin = 16.0
+    private let buttonTrailingMargin = 16.0
+    private let betweenButtonMargin = 8.0
+
+    // Define all button count to calculate holizontal position for capsule rectangle
+    private var buttonsCount: Int {
+        [DroidKaigi2023Day]
+            .fromKotlinArray(DroidKaigi2023Day.values())
+            .count
```

Comment:
Maybe `DroidKaigi2023Day.values().size` is useful 👀

Reply:
@ry-itto

Thank you for review!
It seems to fix `buttonsCount` property as below👍

```
private var buttonsCount: Int {
    Int(DroidKaigi2023Day.values().size)
}
```